### ISSUE TITLE
Experiencing 5 errors  of not found in 'three' from three-vrm.module

### ISSUE THREAD

**Issue by AjionCorp** [pixiv/three-vrm#914]
ERROR in ./node_modules/@pixiv/three-vrm/lib/three-vrm.module.js 3422:11-30
export 'GammaEncoding' (imported as 'THREE') was not found in 'three' 
export 'RGBM7Encoding' (imported as 'THREE') was not found in 'three'
export 'RGBM16Encoding' (imported as 'THREE') was not found in 'three'
export 'RGBDEncoding' (imported as 'THREE') was not found in 'three'
export 'GammaEncoding' (imported as 'THREE') was not found in 'three'

This error occurs starting 0.6.8 and up, including the latest release. It looks like three, removed these 5 types from export, but three-vrm is still using them.

Suggestions on how to bypass this?

**Comment by 0b5vr** [comment]
Mhm, I have written a compat code but it's not working as intended it seems.
I'm going to fix this.

You can use pre-r135 for now I believe.

See: https://github.com/pixiv/three-vrm/pull/899

**Comment by AjionCorp** [comment]
Thank you, i am on VSC and typescript/webpack reactapp, so it drilled down everywhere for possible errors. Going to try with pre-r137, i have a feeling only the v1.0.0 or 0.6.9 is going to fix this

**Comment by 0b5vr** [comment]
@AjionCorp shipped @ v0.6.9 ! Let me know if it's still not working properly

**Comment by grumbeard** [comment]
Hi guys, I'm still seeing export errors for 'RGBM7Encoding' and 'RGBM16Encoding' (using @pixiv/three-vrm v0.6.9 with three v0.137.5)
```
ERROR in ./node_modules/@pixiv/three-vrm/lib/three-vrm.module.min.js 796:11-26
export 'RGBM7Encoding' (imported as 'e') was not found in 'three'
...
ERROR in ./node_modules/@pixiv/three-vrm/lib/three-vrm.module.min.js 799:11-27
export 'RGBM16Encoding' (imported as 'e') was not found in 'three'
```
